pkg/api/v1beta3/types.go

```diff
@@ -586,6 +586,8 @@ type PodSpec struct {
 	// the the scheduler simply schedules this pod onto that host, assuming that it fits
 	// resource requirements.
 	Host string `json:"host,omitempty" description:"host requested for this pod"`
+	// Optional: Default to false.
+	HostNetworking bool `json:"host_networking,omitempty description:"whether or not the container runs in the network namespace of the host; defaults to false; cannot be updated""`
```

Comment:
description: It affects the whole pod, not just an individual container.

json: We use camelCase, not underscores.

I suggest calling it `HostNetwork`/`hostNetwork` (drop the -ing).

Have you thought about the impact on any specified container ports?
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go#L296

Should we ban them if hostNetwork is true? Or should we require hostPort == containerPort (which we could/would also set by default)?